autotst/reaction.py

```diff
@@ -927,7 +927,46 @@ def get_bonds(self):
             self.get_rdkit_mol()
             test_conf._rdkit_molecule = self._pseudo_geometry
         test_conf._ase_molecule = self.ase_molecule
-        return test_conf.get_bonds()
+
+    def get_bonds(self):
+        """
+        A method for identifying all of the bonds in a conformer
+        """
+        bond_list = []
+        for bond in self._pseudo_geometry.GetBonds():
+            bond_list.append((bond.GetBeginAtomIdx(), bond.GetEndAtomIdx()))
+
+        bonds = []
+        for index, indices in enumerate(bond_list):
+            i, j = indices
+
+            length = self.ase_molecule.get_distance(i, j)
+            center = False
+
+            supported_labels = []
+            if self.reaction_family.lower() in [
+                'h_abstraction',
+                'r_addition_multiplebond',
+                'intra_h_migration']:
+                supported_labels = ['*1', '*2', '*3']
```

Comment:
Instead of hard-coding supported labels for specific families, we could try getting the reaction center labels from the reaction recipe (rmgpy.data.kinetics.family.KineticsFamily.forward_recipe.actions).  This should make it easier when we add new families to autotst. Looks like this for intra_H_migration:
```
[['BREAK_BOND', '*2', 1, '*3'],
 ['FORM_BOND', '*1', 1, '*3'],
 ['GAIN_RADICAL', '*2', '1'],
 ['LOSE_RADICAL', '*1', '1']]
```

Reply:
Good point, I'm updating the code to use actions now for both this section and your next comment.